titleize/titleize.js

```diff
@@ -1,6 +1,24 @@
+const isCharacterAllowed = previousCharacter => previousCharacter === '' || previousCharacter === ' ' || previousCharacter === '-';
```

Comment:
This can be easily refactored without using the same variable name again and again. A simple can be `const isCharacterAllowed = char => ['', ' ', '-'].includes(char);`.

Reply:
Also `previousCharacter` does not make sense with respect to the function name `isCharacterAllowed`. As it's a pure function, it's local variables should be in accordance with its function.